doc/classes/DisplayServer.xml

```diff
@@ -1511,6 +1511,7 @@
 			<param index="0" name="window_id" type="int" default="0" />
 			<description>
 				Returns left margins ([code]x[/code]), right margins ([code]y[/code]) and height ([code]z[/code]) of the title that are safe to use (contains no buttons or other elements) when [constant WINDOW_FLAG_EXTEND_TO_TITLE] flag is set.
+				[b]Note:[/b] On Linux amd Windows, this always returns (0, 0) since the window buttons are not displayed by the OS when [constant WINDOW_FLAG_EXTEND_TO_TITLE] flag is set.
```

Comment:
```suggestion
				[b]Note:[/b] On Linux and Windows, this method always returns [code]Vector2(0, 0)[/code], since the decoration buttons are not displayed by the OS when the [constant WINDOW_FLAG_EXTEND_TO_TITLE] flag is set.
```
or (I do prefer this personally)
```suggestion
				[b]Note:[/b] On Linux and Windows, this method always returns [constant Vector2.ZERO], since the decoration buttons are not displayed by the OS when the [constant WINDOW_FLAG_EXTEND_TO_TITLE] flag is set.
```

Not sure if these are called "decoration buttons" but this is the term you used in another description.